src/wp-includes/media.php

```diff
@@ -1903,6 +1910,44 @@ function wp_filter_content_tags( $content, $context = null ) {
 	return $content;
 }
 
+/**
+ * Adds a fallback mechanism to replace webp images with jpeg alternatives on older browsers.
+ *
+ * @since 6.1.0
+ */
+function wp_wepb_fallback() {
+
+	ob_start();
+	$suffix     = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
+	$script_src = includes_url( "js/webp-fallback$suffix.js" );
+
+	?>
+	( function( d, i, s, p ) {
+		s = d.createElement( s );
+		s.src = '<?php echo esc_url_raw( $script_src ); ?>';
+
+		i = d.createElement( i );
+		i.src = p + 'jIAAABXRUJQVlA4ICYAAACyAgCdASoCAAEALmk0mk0iIiIiIgBoSygABc6zbAAA/v56QAAAAA==';
+		i.onload = function() {
+			i.onload = undefined;
+			i.src = p + 'h4AAABXRUJQVlA4TBEAAAAvAQAAAAfQ//73v/+BiOh/AAA=';
+		};
+
+		i.onerror = function() {
+			d.body.appendChild( s );
+		};
+	} )( document, 'img', 'script', 'data:image/webp;base64,UklGR' );
+	<?php
+	$javascript = ob_get_clean();
```

Comment:
While this works well for the Performance Lab plugin, we should follow WP core principles here closely now that it gets towards a core merge: Instead of using an output buffer and putting this script inside a PHP file, it should live within its own JavaScript file and then be retrieved here using `file_get_contents()`, similar to WP core's `wp-emoji-loader.js`. So I would suggest moving this code to a file like `wp-image-mime-fallback-loader.js`. This will also give us the regular JS linting routine on that file. Also: Please change the variable names in that file to be more human-readable. We only used these really short names here since there was no minification running on this, but for core, as we move this into a proper JS file that will also get minified, we should write human-friendly code.

The one dynamic part (the script source of the real fallback script) could then be added here in a global like `_wpImageMimeFallbackSettings`, similar to WP core's `_wpemojiSettings`.

Please review the implementation of WP core's `_print_emoji_detection_script()`, I think we should follow how they do it there in this PR.

Reply:
@felixarntz Do we needs to update similar approach in Performance Lab plugin?

Reply:
@mukeshpanchal27 Thank you, for the Performance Lab plugin this is fine. Of course we could change it, but I don't think it's worth it. What we're doing in the plugin is not necessarily wrong, it's just different than how core does it. So only for the WordPress core merge we should make sure to follow established core development patterns.